src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;

```

Comment:
'fruit' is a very general name for a variable, consider renaming it to something more descriptive, like 'fruitName'.